testnets/_IBC/archwaytestnet-osmosistestnet.json

```diff
@@ -2,29 +2,59 @@
   "$schema": "../ibc_data.schema.json",
   "chain_1": {
     "chain_name": "archwaytestnet",
-    "client_id": "07-tendermint-77",
-    "connection_id": "connection-73"
+    "client_id": "07-tendermint-121",
+    "connection_id": "connection-120"
   },
   "chain_2": {
     "chain_name": "osmosistestnet",
-    "client_id": "07-tendermint-1195",
-    "connection_id": "connection-1101"
+    "client_id": "07-tendermint-3459",
+    "connection_id": "connection-3027"
   },
   "channels": [
     {
       "chain_1": {
-        "channel_id": "channel-58",
+        "channel_id": "channel-225",
         "port_id": "transfer"
       },
       "chain_2": {
-        "channel_id": "channel-3938",
+        "channel_id": "channel-7779",
         "port_id": "transfer"
       },
       "ordering": "unordered",
       "version": "ics20-1",
       "tags": {
         "status": "live"
       }
+    },
+    {
+      "chain_1": {
+        "channel_id": "*",
```

Comment:
These connections with no property channel IDs or ports are concerning.
I'm not comfortable approving it because I don't understand it

Reply:
Hi @JeremyParish69 The reason of this is we are developing smart contract that will create channel for ICA and ICQ, so the channel id will be dynamic and the port id on archway side will be dynamic also. 
Actually Archway chain team is fine with that.